wp-web-push/wp-web-push-main.php

```diff
@@ -5,9 +5,14 @@
 
 class WebPush_Main {
   private static $instance;
+  public static $ALLOWED_TRIGGERS;
 
   public function __construct() {
     add_action('wp_head', array($this, 'add_manifest'));
+    self::$ALLOWED_TRIGGERS = array(
+      array('text' => __('New Post'), 'key' => 'new-post', 'enable_by_default' => true, 'hook' => 'transition_post_status', 'action' => 'send_new_post')
```

Comment:
There's already a `transition_post_status` hook defined. Instead of defining a new function (`send_new_post`), simply use the already existing function (`on_transition_post_status`).